pkg/api/label_values.go

```diff
@@ -12,15 +12,23 @@ import (
 	"net/http"
 )
 
-func LabelValues(queriable *query.Queryable) http.Handler {
-	hf := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+func LabelValues(conf *Config, queryable *query.Queryable) http.Handler {
+	hf := labelValues(queryable)
+	if conf.AllowedOrigin != nil {
+		hf = corsWrapper(conf, hf)
+	}
```

Comment:
1. how come this branch is repeated around each call to `corsWrapper` instead of being within `corsWrapper`?
2. How come each API function calls `corsWrapper` instead of it wrapping the API function like we do for `timeHandler`?